### PR TITLE
modified tooltip content for ldap verification

### PR DESCRIPTION
**Pull request by raymondz1**
### What this PR does / why we need it
For TEST USER NAME, the tooltip should indicate this is the CN attribute for the user, and for TEST GROUP NAME, the tooltip should indicate that this is the OU attribute for the group.
### Which issue(s) this PR fixes

Fixes https://github.com/vmware-tanzu/tanzu-framework/issues/1269

### Release note

```release-note

```

### PR Checklist

- [ ] Squash the commits into one or a small number of logical commits
      
- [ ] Use good commit [messages](https://github.com/vmware-tanzu/tanzu-framework/blob/main/CONTRIBUTING.md)
- [ ] Ensure PR contains terms all contributors can understand and links all contributors can access

### CODE REVIEW

File: pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html [hunk 1]
```diff
@@ -594,7 +594,7 @@ <h3 class="modal-title">Verify LDAP Configuration</h3>
                                             <clr-icon aria-live="assertive" a11yTooltipTrigger shape="info-circle" size="22">
                                             </clr-icon>
                                             <clr-tooltip-content clrPosition="top-right" clrSize="lg" *clrIfOpen>
-                                                <span>A user name used to test LDAP user search functionality.</span>
+                                                <span>A user name used to test LDAP user search functionality; can only be searched by CN.</span>
```
Comment (miclettej) [inline pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html:597]: I think in this case we can simplify to:
Enter the CN attribute for the user to test LDAP user search functionality.

Below on line 601, let's change the placeholder to CN

File: pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html [hunk 2]
```diff
@@ -610,7 +610,7 @@ <h3 class="modal-title">Verify LDAP Configuration</h3>
                                             <clr-icon aria-live="assertive" a11yTooltipTrigger shape="info-circle" size="22">
                                             </clr-icon>
                                             <clr-tooltip-content clrPosition="top-right" clrSize="lg" *clrIfOpen>
-                                                <span>A group name used to test LDAP group search functionality</span>
+                                                <span>A group name used to test LDAP group search functionality; can only be searched by OU.</span>
```
Comment (miclettej) [inline pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html:613]: I think in this case we can simplify to:
Enter the OU attribute for the group to test LDAP group search functionality.

Below on line 617, let's change the placeholder to OU

File: pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html [hunk 3]
```diff
@@ -594,11 +594,11 @@ <h3 class="modal-title">Verify LDAP Configuration</h3>
                                             <clr-icon aria-live="assertive" a11yTooltipTrigger shape="info-circle" size="22">
                                             </clr-icon>
                                             <clr-tooltip-content clrPosition="top-right" clrSize="lg" *clrIfOpen>
-                                                <span>A user name used to test LDAP user search functionality.</span>
+                                                <span>Enter the CN attribute for the user to test LDAP user search functionality.</span>
                                             </clr-tooltip-content>
                                         </clr-tooltip>
                                     </label>
-                                    <input clrInput formControlName="testUserName" placeholder="userid" name="testUserName" />
+                                    <input clrInput formControlName="testUserName" placeholder="CN" name="testUserName" />
```
Comment (sabbey37) [inline pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html:601]: Could we make the placeholder `CN attribute` something like `Test User` or `User's CN Attribute`?

File: pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html [hunk 4]
```diff
@@ -610,11 +610,11 @@ <h3 class="modal-title">Verify LDAP Configuration</h3>
                                             <clr-icon aria-live="assertive" a11yTooltipTrigger shape="info-circle" size="22">
                                             </clr-icon>
                                             <clr-tooltip-content clrPosition="top-right" clrSize="lg" *clrIfOpen>
-                                                <span>A group name used to test LDAP group search functionality</span>
+                                                <span>Enter the OU attribute for the group to test LDAP group search functionality.</span>
                                             </clr-tooltip-content>
                                         </clr-tooltip>
                                     </label>
-                                    <input clrInput formControlName="testGroupName" placeholder="people" name="testGroupName" />
+                                    <input clrInput formControlName="testGroupName" placeholder="OU" name="testGroupName" />
```
Comment (sabbey37) [inline pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html:617]: Could we make the placeholder OU attribute something like `testGroup` or `Group's OU Attribute`?

**Review by sabbey37 — COMMENTED** [review]
Thanks for fixing this, @raymondz1 and sorry for the confusion!

**Review by miclettej — APPROVED** [review]

**Review by sabbey37 — APPROVED** [review]

**Pull request merged**